README.rst

```diff
@@ -5,34 +5,50 @@ Internal and file-IO based JYU-LB engine-wrappers for the SimPhoNy framework.
 
 .. image:: https://travis-ci.org/simphony/simphony-jyulb.svg?branch=master
     :target: https://travis-ci.org/simphony/simphony-jyulb
-      :alt: Build status
 
 .. image:: https://coveralls.io/repos/simphony/simphony-jyulb/badge.svg
    :target: https://coveralls.io/r/simphony/simphony-jyulb
-      :alt: Test coverage
 
 Repository
 ----------
 
-simphony-jyulb is hosted on github: https://github.com/simphony/simphony-jyulb
+Simphony-jyulb is hosted on github: https://github.com/simphony/simphony-jyulb
 
 Requirements
 ------------
- - `simphony-common`_ >= 0.1.1 
+- `simphony-common`_ >= 0.1.1
 
 .. _simphony-common: https://github.com/simphony/simphony-common
 
+- `cython`_
+
+.. _cython: https://pypi.python.org/pypi/Cython/
+
+- `JYU-LB`_ >= 0.1.1
+
+.. _JYU-LB: https://github.com/simphony/JYU-LB
+
 .. note::
   simphony-jyulb uses keywords (COLLISION_OPERATOR, REFERENCE_DENSITY, GRAVITY,
   FLOW_TYPE, EXTERNAL_FORCING) which currently do not belong to the CUBA-keywords.
 
   simphony-jyulb treats the BC data component as a dummy variable (currently there
-  is no SimPhoNy definition for BC).  
+  is no SimPhoNy definition for BC).
 
 Installation
 ------------
 
-The package requires python 2.7.x. Installation is based on setuptools::
+The package requires python 2.7.x, cython and JYU-LB header files along with
+`simphony-common` package to build and install cython extension. To get the
```

Comment:
Why only the JYU-LB header files? I mean, for compiling the cython extensions the JYU-LB source files are needed ...